src/utilities/htmlElements.utils.ts

```diff
@@ -0,0 +1,87 @@
+export const getTextareaHeight = (textareaElement: HTMLTextAreaElement): number => {
+	if (!textareaElement) return 0;
+
+	const copyFullStyle = (source: HTMLTextAreaElement, target: HTMLTextAreaElement) => {
+		const computed = window.getComputedStyle(source);
+		for (const prop of computed as any as string[]) {
+			try {
+				const value = computed.getPropertyValue(prop);
+				if (value) {
+					target.style.setProperty(prop, value, computed.getPropertyPriority(prop));
+				}
+			} catch {
+				// Some properties may throw (rare); ignore safely
```

Comment:
Silent failure is intentional - some CSS properties throw expected exceptions when copied between elements. Logging these benign failures would create log noise without providing actionable information.